openbas-front/src/admin/components/common/Context.ts

```diff
@@ -16,6 +16,8 @@ export type PermissionsContextType = {
 
 export type ArticleContextType = {
   previewArticleUrl: (article: FullArticleStore) => string;
+  fetchChannelsUrl: () => string;
+  fetchDocumentsUrl: () => string;
```

Comment:
Why not follow the existing standard and have a fetchChannels & fetchDocuments direclty ?

Reply:
Because this way, the URL is contextual and derived from the context of either a scenario or a simulation. This is what we came up with @guillaumejparis but I'm not an expert, so let's talk about it?

Reply:
Fixed by creating dedicated fetch methods for simulation and scenario contexts.